client/e2e/support/add-user.po.ts

```diff
@@ -0,0 +1,47 @@
+export interface TestUser {
```

Comment:
Shouldn't an interface like this be in the production code and pulled from there? Otherwise we're basically defining what a `User` object is in (at least) two places.

Reply:
This has been done now.